tools/owl_templates/compile_templates_helpers.cjs

```diff
@@ -0,0 +1,92 @@
+const fs = require("fs");
+const path = require("path");
+const { getParsedOwlTemplateBundle } = require("../bundle_xml/bundle_xml_templates.cjs");
+
+const TEMPLATE_FILE_PATH = path.join(__dirname, "./_compiled/owl_compiled_templates.cjs");
+
+function importOwl() {
+  // Owl need some web globals to work properly. Import them from JSDOM if they aren't already.
```

Comment:
I would hint somewhere, either in the function name or the comment that it's for node js